lib/smoke/get-selenium-grid.js

```diff
@@ -68,14 +68,6 @@ module.exports = async (browserName, isLocal) => {
 			user: process.env.BROWSERSTACK_USER,
 			key: process.env.BROWSERSTACK_KEY
 		};
-	} else if(process.env.SAUCE_USER && process.env.SAUCE_KEY) {
```

Comment:
Could we change the name of this file if it's no longer related to Selenium?

Reply:
It _kind of_ is still related to selenium. But calling it something browserstack definitely makes more sense.

Reply:
Ahh yes, I was confusing Saucelabs and Selenium :man_facepalming: